app/models/task.py

```diff
@@ -1,5 +1,29 @@
-from sqlalchemy.orm import Mapped, mapped_column
+from sqlalchemy.orm import Mapped, mapped_column, relationship
+from sqlalchemy import ForeignKey
 from ..db import db
+from datetime import datetime
+from typing import Optional
 
 class Task(db.Model):
     id: Mapped[int] = mapped_column(primary_key=True, autoincrement=True)
+    title: Mapped[str]
+    description: Mapped[str]
+    completed_at : Mapped[datetime | None] = mapped_column(nullable = True, default = None)
```

Comment:
If you use `[X|None]` syntax, SQLAlchemy is able to infer that this is a nullable field so you do not also need to include `mapped_column` with `nullable=True`.

Nitpick: You have an extra whitespace before the colon that needs to be removed.
```suggestion
    completed_at: Mapped[datetime | None]
```